src/Dommel/Cache.cs

```diff
@@ -20,19 +20,19 @@ internal enum QueryCacheType
 
     internal struct QueryCacheKey : IEquatable<QueryCacheKey>
     {
-        public QueryCacheKey(QueryCacheType cacheType, ISqlBuilder sqlBuilder, MemberInfo memberInfo)
+        public QueryCacheKey(QueryCacheType cacheType, ISqlBuilder sqlBuilder, Type entityType)
```

Comment:
What's the benefit of using `Type` here over `MemberInfo`?

Reply:
This change was made while debugging and would probably have no effect on the changes in Resolver.cs. Feel free to remove that one! 😊

Reply:
Could you remove this change from this PR? Then I can merge it.

Reply:
Sorry, will get right on it!

Reply:
Done, and done!